src/main/java/com/opinta/dto/ShipmentDto.java

```diff
@@ -16,13 +18,9 @@
     private long recipientId;
     @EnumString(source = DeliveryType.class)
     private DeliveryType deliveryType;
-    private float weight;
-    private float length;
-    private float width;
-    private float height;
-    private BigDecimal declaredPrice;
-    private BigDecimal price;
     private BigDecimal postPay;
     @Size(max = 255)
     private String description;
+    private List<ParcelDto> parcels = new ArrayList<>();
+    private BigDecimal price;
```

Comment:
I rewrote a class and on dullness I have changed an order of fields. Already made changes.